package.json

```diff
@@ -13,18 +13,29 @@
     "puppeteer-cluster": "^0.21.0"
   },
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "build": "tsc"
   },
   "files": [
     "src",
     "types",
     "!src/*.spec.js"
   ],
   "devDependencies": {
+    "@babel/core": "^7.16.0",
+    "@babel/preset-env": "^7.16.0",
+    "@babel/preset-typescript": "^7.16.0",
+    "@types/handlebars": "^4.1.0",
+    "@types/jest": "^27.0.2",
+    "@types/node": "^16.11.6",
+    "@types/puppeteer": "^5.4.4",
+    "@types/puppeteer-core": "^5.4.0",
+    "babel-jest": "^27.3.1",
```

Comment:
I think you can remove `babe-jest` from the dependencies because it is already a dependencies of `jest`. This should simplify the yarn.lock and avoid version conflict :)
```suggestion
```

Reply:
I followed this documentation https://jestjs.io/docs/getting-started#using-typescript which ask to follow this one https://jestjs.io/docs/getting-started#using-babel. It tells to install babel-jest :( 

I'll give it a try 👍

Reply:
Works like a charm 👌